README.md

```diff
@@ -1181,4 +1181,4 @@ See [CONTRIBUTING](CONTRIBUTING.md).
 
 ## Copyright and License
 
-Copyright (c) 2012-2014 Tim Vandecasteele and contributors. See [LICENSE.txt](LICENSE.txt) for details.
+Copyright (c) 2012-2016 Tim Vandecasteele and contributors. See [LICENSE.txt](LICENSE.txt) for details.
```

Comment:
But copyright noice that says it's his copyright till 2015 and not after seems to contradict what @platfrms is saying no? Either way this is just my opinion, do what you need to do :)